src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ToolStripCustomTypeDescriptorTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class ToolStripCustomTypeDescriptorTests : IDisposable
+{
+    private readonly ToolStrip _toolStrip;
+    private readonly ToolStripCustomTypeDescriptor _descriptor;
+
+    public ToolStripCustomTypeDescriptorTests()
+    {
+        _toolStrip = new();
+        _descriptor = new(_toolStrip);
+    }
+
+    public void Dispose()
+        => _toolStrip.Dispose();
```

Comment:
```suggestion
    public void Dispose() => _toolStrip.Dispose();
```

This is a short line, it's more readable if it fits into a single line.